apps/web/cypress/tests/integrations-list-page.spec.ts

```diff
@@ -291,63 +297,70 @@ describe('Integrations List Page', function () {
   });
 
   it('should show the select provider sidebar', () => {
-    cy.intercept('*/integrations', async () => {
-      await new Promise((resolve) => setTimeout(resolve, 1000));
-    }).as('getIntegrations');
-    cy.intercept('*/environments').as('getEnvironments');
-
-    cy.visit('/integrations');
-    cy.location('pathname').should('equal', '/integrations');
-
-    cy.getByTestId('add-provider').should('be.disabled').contains('Add a provider');
-
-    cy.wait('@getIntegrations');
-    cy.wait('@getEnvironments');
-
-    cy.getByTestId('add-provider').should('be.enabled').click();
-
-    cy.location('pathname').should('equal', '/integrations/create');
-    cy.getByTestId('select-provider-sidebar').should('be.visible').as('selectProviderSidebar');
-
-    cy.get('@selectProviderSidebar').getByTestId('sidebar-close').should('be.visible');
-    cy.get('@selectProviderSidebar').contains('Select a provider');
-    cy.get('@selectProviderSidebar').contains('Select a provider to create instance for a channel');
-    cy.get('@selectProviderSidebar')
-      .find('input[type="search"]')
-      .should('have.attr', 'placeholder', 'Search a provider...');
-
-    cy.get('@selectProviderSidebar').find('[role="tablist"]').as('channelTabs');
-    cy.get('@channelTabs').find('[data-active="true"]').contains('In-App');
-    cy.get('@channelTabs').contains('In-App');
-    cy.get('@channelTabs').contains('Email');
-    cy.get('@channelTabs').contains('Chat');
-    cy.get('@channelTabs').contains('Push');
-    cy.get('@channelTabs').contains('SMS');
-
-    cy.getByTestId('providers-group-in_app').contains('In-App').as('inAppGroup');
-    cy.getByTestId('providers-group-email').contains('Email').as('emailGroup');
-    cy.getByTestId('providers-group-chat').contains('Chat').as('chatGroup');
-    cy.getByTestId('providers-group-push').contains('Push').as('pushGroup');
-    cy.getByTestId('providers-group-sms').contains('SMS').as('smsGroup');
+    cy.task('deleteProvider', {
```

Comment:
Need to delete in app provider here because we create both on init session so for that option to be visibile we need to delete them here.